package.json

```diff
@@ -49,16 +51,17 @@
     "metalsmith-metadata": "0.0.2",
     "metalsmith-permalinks": "0.4.0",
     "metalsmith-prism": "2.1.1",
-    "metalsmith-stylus": "1.0.0",
+    "metalsmith-stylus": "^2.0.0",
```

Comment:
`metalsmith-stylus` is already at version `2.0.0` on master.
Some other dependencies updated here are also already updated on master.